packages/office-ui-fabric-react/src/components/MarqueeSelection/MarqueeSelection.base.tsx

```diff
@@ -74,6 +75,10 @@ export class MarqueeSelectionBase extends BaseComponent<IMarqueeSelectionProps,
     if (this._autoScroll) {
       this._autoScroll.dispose();
     }
+    if (this._evaluateSelection) {
+      delete this._evaluateSelection;
+    }
+    this._events.dispose();
```

Comment:
Not necessary since this component extends `BaseComponent`. See my similar comment https://github.com/OfficeDev/office-ui-fabric-react/pull/9439/files#r293216112.

Reply:
I removed this.